http/util/getCommonMiddleware.js

```diff
@@ -0,0 +1,7 @@
+const koaBodyParser = require('koa-bodyparser')()
+
+module.exports = function getCommonMiddleware () {
+  return [
+    koaBodyParser
```

Comment:
Not every route will need this (ex. health checks). Not a huge deal, though.

Reply:
Yeah, but we talked about this in person and determined that this is not a big deal. Right?

Reply:
Yep, nbd.